### ISSUE TITLE
Intent to implement: add built-in vendor configuration for ShinyStat analytics

### ISSUE THREAD

**Issue by shinystat** [ampproject/amphtml#2557]
## Objective

To implement ShinyStat vendor configuration.
## Proposed Vendor Configuration

```
<amp-analytics type="shinystat">
<script type="application/json">
{
    "vars": {
        "account": "foobar"
        "page": "Homepage"
    }
}
</script>
</amp-analytics>
```

Pull request number #2556.

**Comment by adelinamart** [comment]
What's the status here? Thanks.

**Comment by jridgewell** [comment]
Stalled in https://github.com/ampproject/amphtml/pull/2556.

**Comment by mrjoro** [comment]
It looks like this was fixed in #11483, so I'm closing this.  Please reopen if needed.
